File: problems/prob06/prob6.js
var rand_print = function (rand_num){
	var greaterThan = []
	var lessThan = []
	if (rand_num > 40) {
		for (var i = 40; i <= rand_num; i++) {
			if (i % 2 !== 0){
				greaterThan.push(i)
			}
		}
		return greaterThan.reverse()
	} else {
		for (var i = rand_num; i < 40; i++) {
			if (i % 2 !== 0) {
				lessThan.push(i)
			}
		}
		return lessThan.reverse()
	}
}

var rand_num = (Math.floor((Math.random() * 100)))
document.write("The number is " + rand_num + "<br/>")
document.write(rand_print(rand_num))

File: problems/prob06/prob6.py
from random import randrange


def generate_list(num):
    greater_than = []
    less_than = []
    if num > 40:
        for x in range(40, num + 1):
            if x % 2 != 0:
                greater_than.append(x)
        return greater_than
    else:
        for x in range(0, num + 1):
            if x % 2 != 0:
                less_than.append(x)
        return less_than


def reverse(lst):
    reverse_list = []
    for i in reversed(lst):
        reverse_list.append(i)
    return reverse_list


rand_num = randrange(100)
print("The number is {0}".format(rand_num))
print(reverse(generate_list(rand_num)))
